container/container.go

```diff
@@ -59,4 +59,7 @@ type ContainerHandler interface {
 	// Start starts any necessary background goroutines - must be cleaned up in Cleanup().
 	// It is expected that most implementations will be a no-op.
 	Start()
+
+	//Name of handler
+	String() string
```

Comment:
I'd rather call it `Name` since I'd expect a little more information from a `String` method. 

godoc nit, space missing and should start with the function name